Added method to clear specific object store on a DB

sombhattacharyya1983 opened the pull request:
Added method to clear specific object store on a DB

Polarisation commented:
Not convinced about adding this as it stands, since the import & export functions operate on the whole DB not a single object store. If those also had an option to work on a single object store then I think it would be a good addition.

However, it would be better to provide the store name as an optional argument to the existing methods. This would keep the size of the library small.

sombhattacharyya1983 commented:
> Not convinced about adding this as it stands, since the import & export functions operate on the whole DB not a single object store. If those also had an option to work on a single object store then I think it would be a good addition.
> 
> However, it would be better to provide the store name as an optional argument to the existing methods. This would keep the size of the library small.

I agree. I have made the changes to include the 'store name' as an optional parameter. Kindly check if it helps. Thanks.

Polarisation commented:
Thank you @sombhattacharyya1983 

Before I could accept this PR I'd want to see all of the following addressed:

- Conflicts resolved
- All the methods would need to support an optional parameter for a specific object store. I don't see much value in having an option only on the `clearDatabase` method.
- The default `storeName` value should be `null` not `'nothing'` because there could be an object store named "nothing"
- I think the code can be made a bit more concise by populating the array of object store names with the single item provided via the `storeName` parameter
- The README needs to be updated to document the new parameter.

sombhattacharyya1983 commented:
@Polarisation I should be able to iron those out over the weekend. Thanks.

sombhattacharyya1983 commented:
@Polarisation Can you please review once when you have the time. I have addressed the points as mentioned by you. Wanted you to have a quick look so I can squash the commits in one.
